packages/model/DimensionMetadata.ts

```diff
@@ -65,8 +67,18 @@ export function DimensionMetadataCollectionMixin<Base extends Constructor>(Resou
 
 DimensionMetadataCollectionMixin.appliesTo = cc.DimensionMetadataCollection
 
+export const Error = {
+  MissingMeasureDimension: 'MissingMeasureDimension',
+} as const
+
+export const createNoMeasureDimensionError = Thing.fromPointer({
+  identifierLiteral: Error.MissingMeasureDimension,
+  description: 'No Measure dimension defined',
+})
+
 export const createCollection = initializer<DimensionMetadataCollection>(DimensionMetadataCollectionMixin, {
   types: [cc.DimensionMetadataCollection],
+  [schema.error.value]: [createNoMeasureDimensionError],
```

Comment:
If you wonder about the lockfile updates, they are to bring in this feature where properties can be set using factory. `createNoMeasureDimensionError` is a `pointer => Resource` function so that resources can be initialized lazily, with the actual node being assigned only when the setter is used